packages/api/src/command/medical/patient/consolidated-get.ts

```diff
@@ -294,7 +289,22 @@ export async function getConsolidated({
 
     analytics(defaultAnalyticsProps);
 
+    const startedAt = new Date();
+    const initialBundleLength = bundle.entry?.length;
     bundle = deduplicateSearchSetBundle(bundle);
+    const finalBundleLength = bundle.entry?.length;
+
+    const deduplicationAnalyticsProps = {
+      distinctId: patient.cxId,
+      event: EventTypes.fhirDeduplication,
+      properties: {
+        patientId: patient.id,
+        initialBundleLength,
+        finalBundleLength,
+        duration: elapsedTimeFromNow(startedAt),
+      },
```

Comment:
❤️ 
lets add an item to the release plan to create a new view in Posthog. I can help with that if you havent done it before